synthnn/core/accelerated_musical_network.py

```
@@ -49,6 +49,18 @@ def __init__(self, name: str = "accelerated_musical",
         self._device_connections = None
         
         print(f"AcceleratedMusicalNetwork initialized with {type(self.backend).__name__}")
+
+    def _get_mode_intervals(self, mode: str) -> List[float]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)